package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-cosmosdb",
-    "version": "0.20.0",
```

Comment:
In the future, we should add the alpha suffix after we release.

Reply:
We did, this PR shows 0.20.0 because it branches off the commit where 0.20.0 is released.